README.md

```diff
@@ -1,18 +1,55 @@
-# apollo-cli
+# rover-cli
 > ✨🤖 🐶 the new CLI for apollo
 
 [![Tests](https://github.com/apollographql/apollo-cli/workflows/Tests/badge.svg)](https://github.com/apollographql/apollo-cli/actions?query=workflow%3ATests)
 ![Stability: Experimental](https://img.shields.io/badge/stability-experimental-red)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/1646a37a-eb2b-48e8-b6c9-cd074f02bb50/deploy-status)](https://app.netlify.com/sites/apollo-cli-docs/deploys)
 
-This is the home of the new CLI for Apollo's suite of GraphQL developer productivity tools.
+This is the home of `rover`, the new CLI for Apollo's suite of GraphQL developer productivity tools.
 
-This repo is organized as a [`cargo` workspace], containing two related projects:
+## Command-line options
+
+```
+✨🤖🐶 the new CLI for Apollo
+
+USAGE:
+    rover [OPTIONS] <SUBCOMMAND>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -l, --log <log-level>     [default: debug]  [possible values: error, warn, info,
+                             debug, trace]
+
+SUBCOMMANDS:
+    config     ⚙️  Rover configuration
+    help       Prints this message or the help of the given subcommand(s)
+    partial    *️⃣  Federated schema/graph commands
+    schema     ⏺  Non-federated schema/graph commands
+```
+
+This repo is organized as a [`cargo` workspace], containing several related projects:
 
 - `rover`: a CLI that wraps `apollo` 
 - `apollo`: a library containing functionality to interact with graphs and Apollo services
+- `houston`: utilities fro configuring the `rover` CLI tool
+- `robot-panic`: a fork of [rust-cli/robot-panic] adjusted for `rover` CLI
+- `rover-client`: an HTTP client for making graphql requests for `rover` CLI
+- [`sputnik`]: a crate to aid in collection of anonymous data for Rust CLIs
+- `timber`: a log formatter for `env_logger` and `log` crates
 
 [`cargo` workspace]: https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html
+[rust-cli/robot-panic]: https://github.com/rust-cli/robot-panic
+[`sputnik`]: https://github.com/apollographql/rover/tree/main/crates/sputnik
```

Comment:
should we add links to each of the other sub-crates?

Reply:
sure! done.